### ISSUE TITLE
faced with error in prod env when try to open api doc page  Return value of phpDocumentor\Reflection\TypeResolver::

### ISSUE THREAD

**Issue by shubaivanqbee** [nelmio/NelmioApiDocBundle#1912]
I use

"nelmio/api-doc-bundle": "v4.6.2",

and when try to open in dev env api doc looks correct without error, but when to switched to prod env I faced with strange error

```
Return value of phpDocumentor\Reflection\TypeResolver::parseTypes() must implement interface phpDocumentor\Reflection\Type, null returned

file = "/alert_service/vendor/phpdocumentor/type-resolver/src/TypeResolver.php"
line = {int} 318
```

I calerad cache, so, what it could be ?

**Comment by alshenetsky** [comment]
I think it's phpDocumentator bug: https://github.com/phpDocumentor/TypeResolver/issues/141

**Comment by jaapio** [comment]
Hi, I'm trying to find the issue in our type resolver but I cannot discover any path that would result in an error in this class. 

If you have any information regarding the types that need to be resolved I would be very happy to get that. I think one of the objects that is used to create your api docs contains an unsupported type definition.

**Comment by TrueType** [comment]
added a hint, how the problem may originate in user space code - https://github.com/phpDocumentor/TypeResolver/issues/141

**Comment by jaapio** [comment]
This issue has been fixed in typeresolver v1.6

**Comment by GuilhemN** [comment]
Nice, thanks for the follow up!
Let's close this then :)
